webapp/src/utils/helpers.ts

```diff
@@ -58,6 +59,9 @@ export const parseSearchString = (searchString: string) => {
     postprocessing: convertSearchParams<QueryPostprocessingState>(q, {
       withoutPostprocessing: (s) => s !== null || undefined,
     }),
+    normalizedState: convertSearchParams<QueryNormalizedState>(q, {
```

Comment:
I think I would call that the "confusion matrix state", as it could welcome more options. It also allows to remove the `State` from the variable name, making it uniform with the others, without causing a confusing `normalized.normalized`. Instead, `confusionMatrix.normalized` is quite clear.
Makes sense?
```suggestion
    confusionMatrix: convertSearchParams<QueryConfusionMatrixState>(q, {
```

Reply:
I always guess the name inappropriate :) I should be improving that part

Reply:
Haha! It was not "inappropriate". Don't worry, that's why we do reviews.

Reply:
I see you added `State` to the field name `confusionMatrix`. Could you remove it to keep it uniform with the fields?

Reply:
Right, the reason I added "state" to the suggested name because of the variable names conflict on the file
Here is the conflict
  const { data: { confusionMatrix, normalized } = { confusionMatrix: [] } } =
    getConfusionMatrixEndpoint.useQuery({
      jobId,
      datasetSplitName,
      ...filters,
      ...pipeline,
      ...postprocessing,
      ...confusionMatrixState,
    });

Reply:
I would prefer renaming that variable so it only impact one file. Something like:
```diff
-const { data: { confusionMatrix, normalized } = { confusionMatrix: [] } } =
```

Reply:
yes thats true. I agree